test/src/sdk-harness/test_projects/storage_vec/mod.rs

```diff
@@ -0,0 +1,122 @@
+mod utils;
```

Comment:
As above, if the tests are meant to be in their own directories then they shouldn't be here. I don't get the point of this file. Perhaps this one should be the "utils" where everything is imported from?

If that doesn't work then it makes sense to me to keep each test within its project and then import the utils from 1 file. Atm it seems like you have an empty test in the project itself and then a utils and here you do the tests.

Reply:
The empty test in the project has been removed